java/src/test/java/vw/learner/VWFloatArrayLearnerTest.java

```diff
@@ -0,0 +1,73 @@
+package vw.learner;
```

Comment:
@marko-asplund I was going through this class and I cleaned up some redundant code in here.  There were two more tests, one that didn't assert anything and one that checked if an exception would be thrown which would have be triggered by the one test left.  I'm not too familiar with how LDA works in VW and I'm not sure exactly what this last test is doing.  Can you please advise.